### PR TITLE
docs: update Hire us to directly lead to contact form

### PR DESCRIPTION
**Pull request by kacperkapusciak**
This PR changes the Hire us links in the documentation and README to directly lead to the opened project contact form. Previously it led to the whole Contact page.

### CODE REVIEW

**Review by piaskowyk — APPROVED** [review]

**Review by patrycjakalinska — APPROVED** [review]
🚀

**Comment by tomekzaw** [comment]
Let's also update the link in README.md.

**Review by tomekzaw — CHANGES_REQUESTED** [review]
Let's also update the link in README.md:

https://github.com/software-mansion/react-native-reanimated/blame/main/README.md#L59

**Review by tomekzaw — APPROVED** [review]

**Review by tomekzaw — CHANGES_REQUESTED** [review]
One more:
https://github.com/software-mansion/react-native-reanimated/blob/987e017c999a4ab3054cec9538e53929e133c3fa/packages/docs-reanimated/src/components/HireUsSection/index.tsx#L21

File: packages/docs-reanimated/src/components/HireUsSection/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const HireUsSection = () => {
         <HomepageButton
           backgroundStyling={styles.buttonNavyStyling}
           borderStyling={styles.buttonNavyBorderStyling}
-          href="https://swmansion.com/contact#contact-form"
+          href="https://swmansion.com/contact/projects?utm_source=reanimated&utm_medium=landing-page"
```
Comment (tomekzaw) [inline packages/docs-reanimated/src/components/HireUsSection/index.tsx:21]:
```suggestion
          href="https://swmansion.com/contact/projects?utm_source=reanimated&utm_medium=docs"
```

**Pull request merged**